### DISCUSSION TITLE
Suspect that publishers have a leak

### DISCUSSION THREAD

**Discussion by st-small** [enebin/Aespa#31] [Q&A]
I am trying to use a library and have some questions. If a user captures a photo, it is stored in the Photo app. Then, we suspend the application and open the Photos app. We delete all captured photos and return back. My app shows the last captured photo in the preview, but it should show nothing. I suspect that publishers cache old values and can't get to a nil state.

**Comment by enebin** [comment]
It seems like an issue occurs because the publisher cannot handle the delete event. I've registered it in the Issue, and I will try to resolve it as soon as possible. Please pay attention to [this issue](https://github.com/enebin/Aespa/issues/34), and let me know if you encounter any problems in the future!

**Comment by enebin** [comment]
@st-small It's a bit late and you may have found another solution or another library.... but the feature added in the latest version `0.5.2`:rocket:.
For details, check out this part of the demo codes. 
https://github.com/enebin/Aespa/blob/334e5ab0c36baad87c4631043958839a86c6eaf2/Demo/Aespa-iOS/VideoContentViewModel.swift#L86-L117
